Author: Jim Hodapp <jim.hodapp@canonical.com>
Description: Adjust backporting to specify correct Qt version.
 Adjust the other patches so that they're visible as if they were from the 
 version they appear in upstream too. This makes it possible to simply drop 
 all the patches when switching to Qt 5.6, without affecting apps.
Last-Update: 2015-12-21
Forwarded: no

diff -urN qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/qdeclarativeaudio.cpp 2/qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/qdeclarativeaudio.cpp
--- qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/qdeclarativeaudio.cpp	2015-12-21 07:20:31.000000000 +0000
+++ qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/qdeclarativeaudio.cpp	2015-12-21 07:18:08.000000000 +0000
@@ -290,6 +290,7 @@
         emit playlistChanged();
     }
 
+    m_playlist = NULL;
     m_source = url;
     m_content = m_source.isEmpty() ? QMediaContent() : m_source;
     m_loaded = false;
@@ -556,6 +557,8 @@
     This property holds the playlist used by the media player.
 
     Setting the \l playlist property resets the \l source to an empty string.
+
+    \since 5.6
 */
 
 /*!
diff -urN qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/Video.qml 2/qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/Video.qml
--- qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/Video.qml	2015-12-21 07:20:31.000000000 +0000
+++ qtmultimedia-opensource-src-5.5.1/src/imports/multimedia/Video.qml	2015-12-21 07:18:08.000000000 +0000
@@ -327,6 +327,8 @@
         This property holds the playlist used by the media player.
 
         Setting the \l playlist property resets the \l source to an empty string.
+
+        \since 5.6
     */
     property alias playlist:        player.playlist
 
